intro-to-object-oriented-programming/what-is-oop.md

```diff
@@ -66,12 +66,13 @@ Each class definition will have:
 
 - A class name
   - This name is used as the "data type" name
-- Concept-related **attributes**
+- **Attributes**
   - An attribute is a piece of data
   - A class definition holding an attribute means we expect this class to hold a specific piece of data
-- Concept-related **methods**
-  - A method is a function that describes behavior or action
-- Logic to describe what happens every time the class is used to make an **instance.**
+- **Methods**
+  - A **method** is a function that describes behavior or action of a class
```

Comment:
With the 'of a class' addition, it sounds better to me to pluralize behavior and action
```suggestion
  - A **method** is a function that describes behaviors or actions of a class
```

Reply:
If we go with the shorter version (my vote) disregard this.

Reply:
I don't really understand the plurals here. The idea that one method would have multiple behaviors or actions feels to me like it goes against the Single-Responsibility Principle. But I see how the grammar feels a little funky here.

Reply:
...so I'll just put this sentence back like it originally was :)